servo-build-dependencies/ci.sls

```diff
@@ -0,0 +1,14 @@
+{% from 'common/map.jinja' import root %}
+{% from tpldir ~ '/map.jinja' import sccache %}
+
+sccache:
+  file.managed:
+    - name: {{ sccache.destination }}
+    - source: https://s3.amazonaws.com/rust-lang-ci/rust-ci-mirror/{{ sccache.version }}-sccache-{{ sccache.platform }}
```

Comment:
Just one question - where does this come from? Do we want our own for Servo? cc @edunham